app/views/form/_numeric_output_question.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render partial: "form/guidance/#{question.guidance_partial}" if question.guidance_partial %>
```

Comment:
Might be worth checking whether this would be accepted upstream? @paulrobertlloyd  what do you think?